### ISSUE TITLE
Current pages in nav menus should be indicated with aria-current="page"

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#2387]
The current page is not visually indicated or announced when navigating. This is a problem throughout the app, making it hard for both signed and AT users to orient themselves.

![image.png](https://images.zenhubusercontent.com/620427f3846ebb620d78e582/efe7b569-ef6d-45d3-adf8-6f108ec0141f)

**Comment by thiessenp-cds** [comment]
I have rarely seen this used but in theory it sounds like a helpful way to let a user know where s/he is in the app. My assumption is to use aria-current to highlight where a user is in navigation lists? 

If so, I did a quick scan of our major pages and for each page with a navigation list (screenshots below), the current item in the list that is the active page should have an aria-current="page" and the other items either remove or set aria-current="false". Sounds about right?

The squiggly line is the list, the underline is the current page that should have an aria-current="page".

Note: the Profile page with the dropdown for profile|accounts.., that may be overkill to add an aria-current but I thought I'd include it for your thoughts :)
 
![Screenshot 2023-08-16 at 10 22 06 AM](https://github.com/cds-snc/platform-forms-client/assets/107579368/28bbe7d7-e72b-4ae3-b99d-f7d08f79c19f)
![Screenshot 2023-08-16 at 10 23 47 AM](https://github.com/cds-snc/platform-forms-client/assets/107579368/99ecfa6f-d4ac-4995-86f6-46e0c2ca9662)
![Screenshot 2023-08-16 at 10 24 14 AM](https://github.com/cds-snc/platform-forms-client/assets/107579368/2fc321a5-1aa2-4f6f-9f7f-9f3f16e336a0)
![Screenshot 2023-08-16 at 10 46 29 AM](https://github.com/cds-snc/platform-forms-client/assets/107579368/10927034-34f8-415b-a18f-b19fcf41c54a)

**Comment by connorscarolyns** [comment]
@thiessenp-cds I think it would be helpful. I watched AT users who were unable to identify which page or tab they were on. I didn't realize this wasn't standard practice because I have always included it in my designs. We use visual indications such as bolding or highlighting to show a current state, which tells me we should also include an analogous equivalent for AT users.

**Comment by thiessenp-cds** [comment]
Cool. Let's do this then. 
For things that look like tabs, we could start using Tab semantics for them e.g. Accounts page with All|Active|Deactivated?

**Comment by connorscarolyns** [comment]
> For things that look like tabs, we could start using Tab semantics for them e.g. Accounts page with All|Active|Deactivated?

Yes, that would work.
